monitoring_prometheus/controllers/prometheus_metrics.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2016-2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.http import Controller, route
+from prometheus_client import generate_latest
+
+
+class PrometheusController(Controller):
+    @route('/metrics', auth='public')
```

Comment:
I have two questions on the route.
* The endpoint naming could be more specific something like '/metrics/prometheus' ?
* Would it not be best practice to have some security on the end point if not now at least a todo remark somewhere ?

Reply:
1. By default, a prometheus server will check `/metrics`
2. Firstly, the data available at this endpoint is really low-level, it's only numbers. The only risk there is a DDOS attack on the endpoint but this is already mitigated at the load balancer level. Secondly, adding security here means that we need to develop a custom collector on Prometheus side to be able to authenticate on Odoo before fetching the data and then have a specific configuration for each Odoo instance instead of having a generic one

Reply:
Thanks for your answer